pywbem_mock/_mainprovider.py

```diff
@@ -811,152 +790,247 @@ def remove_namespace(self, namespace):
     #
     ####################################################################
 
-    def EnumerateClasses(self, **params):
-        # pylint: disable=invalid-name
+    def EnumerateClasses(self, namespace, ClassName=None,
+                         DeepInheritance=None, LocalOnly=None,
+                         IncludeQualifiers=None, IncludeClassOrigin=None):
         """
-        Implements a mock server responder for
+        Implements a CIM server responder for
```

Comment:
In this file, I found the following terms for this:
* `CIM server responder`
* `WBEM provider responder`
* `CIM repository server responder`
* `server request responder`
* `mock WBEM provider responder`
* `CIM repository responder`

Reply:
Changed but without the word mock